texastribune/donations
# Changes for Giving Tuesday 2022 (thermometer 🌡)

#### What's this PR do?
This PR sets up the thermometer for the donations page.

#### Why are we doing this? How does it help us?
We are doing this for the Giving Tuesday Drive. 

#### How should this be manually tested?
You should be able to spin up the donations environment and see the thermometer at the top of the donations page. The label should show $20,000. 

#### How should this change be communicated to end users?
N/A

#### Are there any smells or added technical debt to note?
N/A

#### What are the relevant tickets?
https://airtable.com/appyo1zuQd8f4hBVx/tbloNZu8GkM52NKFR/viwBixFZ2aLPg6nXu/recdOMsnT0URDbCQF?blocks=hide 

#### Have you done the following, if applicable:
***(optional: add explanation between parentheses)***


#### TODOs / next steps:
